Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/shopify/use-content.md

# useContent composable

> `useContent` composition API function is responsible, as its name suggests, for interactions with cms pages from your eCommerce. This function returns the following values:

Reviewer: ```suggestion
The `useContent` composition API function allows loading of the CMS pages. This function returns the following values:
```
